.evergreen/config.yml

```diff
@@ -790,6 +769,39 @@ functions:
           python oidc_write_orchestration.py
           python oidc_get_tokens.py
 
+  "bootstrap aws-secrets":
+    - command: ec2.assume_role
+      params:
+        role_arn: ${aws_test_secrets_role}
+    - command: shell.exec
```

Comment:
This command can be removed if we use `add_expansions_to_env` in the next command.

Reply:
By adding `${aws_test_secrets_role}` as an env variable to the next command using `add_expansions_to_env`?

Reply:
No, `ec2.assume_role` creates the expansions.  When you use `add_expansions_to_env`, all of the available expansions are set to environment variables in that shell.  You can then remove the `prepare_aws_connection.sh` handling.  We also don't need the pypy3.7 since we dropped pypy3.7 support

Reply:
We dropped PyPy 3.7 support? We still have a lot of EG variants with it, we should remove those then.

Reply:
If you merge from master, you'll pick up #1343